polling_stations/apps/data_collection/management/commands/import_flintshire.py

```diff
@@ -0,0 +1,45 @@
+from data_collection.management.commands import BaseXpressDemocracyClubCsvImporter
+
+
+class Command(BaseXpressDemocracyClubCsvImporter):
+    council_id = "W06000005"
+    addresses_name = "parl.2019-12-12/Version 2/Democracy_Club__12December2019flint.tsv"
+    stations_name = "parl.2019-12-12/Version 2/Democracy_Club__12December2019flint.tsv"
+    elections = ["parl.2019-12-12"]
+    csv_delimiter = "\t"
+    csv_encoding = "windows-1252"
+    allow_station_point_from_postcode = False
+
+    def station_record_to_dict(self, record):
+        if record.polling_place_id == "5162":
+            # Parish Hall/Neuadd Y Plwyf Halkyn/Helygain
+            record = record._replace(polling_place_postcode="CH8 8BU")
+        if record.polling_place_id == "5237":
+            # Ysgol Y LLan, Whitfield
+            record = record._replace(polling_place_uprn="10013703522")
+        if record.polling_place_id == "10090462071":
```

Comment:
This is not an Xpress polling station ID

Reply:
Thanks for the catch. Now fixed :-)